src/libopensc/card-piv.c

```diff
@@ -3037,6 +3039,18 @@ static int piv_match_card_continued(sc_card_t *card)
 					!(memcmp(card->reader->atr_info.hist_bytes, "Yubikey", 7))) {
 				type = SC_CARD_TYPE_PIV_II_NEO;
 			}
+			/*
+			 * https://csrc.nist.gov/csrc/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp1239.pdf
+			 * lists 2 ATRS with historical bytes:
+			 *   73 66 74 65 2D 63 64 30 38 30
+			 *   73 66 74 65 20 63 64 31 34 34
+			 * will check for 73 66 74
+			 */
+			else if (card->reader->atr_info.hist_bytes_len >= 3 &&
+					!(memcmp(card->reader->atr_info.hist_bytes, "fte", 3))) {
```

Comment:
This still isn't working for me — I believe because this comparison is off by one byte?
`73 66 74` = "sft"
`66 74 65` = "fte"